include/aws/io/retry_strategy.h

```diff
@@ -235,6 +243,16 @@ AWS_IO_API struct aws_retry_strategy *aws_retry_strategy_new_standard(
     struct aws_allocator *allocator,
     const struct aws_standard_retry_options *config);
 
+/**
+ * This retry strategy is used to disable retries.
```

Comment:
add document to mention that the config is optional, it can be null. Since normally, the config will be required to be not null.

Reply:
Thanks, updated.